x-pack/plugins/lens/public/react_embeddable/types.ts

```diff
@@ -378,6 +381,13 @@ export type LensApi = Simplify<
     HasLibraryTransforms<LensRuntimeState> &
     // Let the container know the view mode
     PublishesViewMode &
+    // forward the parentApi, note that will be exposed only if it satisfy the PresentationContainer interface
+    Partial<
+      HasParentApi<
+        PresentationContainer &
+          Partial<Pick<PublishesPanelTitle, 'hidePanelTitle'> & PublishesViewMode>
```

Comment:
Two things here:
1- why the `parentApi` have both the `PresentationContainer` and the `hidePanelTitle` + PublishesViewMode? 
the `apiIsPresentationContainer` function just checks for the `PresentationContainer` property non for the rest.

2- isn't `PublishesViewMode` already available in the API? why does it need to be also exposed from the parentApi? aren't them the same?
```
export function initializeEditApi(
...
 const [viewMode$] = buildObservableVariable<ViewMode>(
    extractInheritedViewModeObservable(parentApi)
  );
```

Reply:
I would have liked to be `unknown` to be honest, but the `buildApi` signature wants a `parentApi` who comply with the `PresentationContainer` interface.
I've removed the other types from it now with [4f839c8](https://github.com/elastic/kibana/pull/202006/commits/4f839c81037589b7a2848f1f91f20049ff3d3ae9)